astroplan/core.py

```diff
@@ -161,6 +163,46 @@ def altaz(self, time, target=None, obswl=None):
                 coordinate = target
             return coordinate.transform_to(altaz_frame)
 
+    def parallactic_angle(self, time, target):
+        '''
+        Calculate the parallactic angle.
+
+        Parameters
+        ----------
+        time : `~astropy.time.Time`
+            Observation time.
+
+        target : `~astroplan.FixedTarget` or `~astropy.coordinates.SkyCoord`
+            Celestial object of interest.
+
+        Returns
+        -------
+        `~astropy.coordinates.Angle`
+            Parallactic angle
+        '''
+
+        if not (hasattr(target, 'ra') or hasattr(target, 'dec') or
+                hasattr(target, 'coord')):
+            raise TypeError('The target must be a coordinate (i.e. a '
+                            'FixedTarget or SkyCoord).')
+
+        if hasattr(target, 'coord'):
+            coordinate = target.coord
+        else:
+            coordinate = target
+
+        # Eqn (14.1) of Meeus' Astronomical Algorithms
+
+        H = (time.sidereal_time('mean', longitude=self.location.longitude)
+             - coordinate.ra).radian
+        q = np.arctan(np.sin(H) /
+                      (np.tan(self.location.latitude.radian)*
+                       np.cos(coordinate.dec.radian) -
+                       np.sin(coordinate.dec.radian)*np.cos(H)))*u.rad
+
+        if q < 0*u.rad:
```

Comment:
I looked into this before I posted the PR and I think what we need to do is subtler than a wrap, because the definition (according to @adrn?) of parallactic angle is bounded on (0,π). `Angle().wrap_at()` [wraps things within 360 deg](https://astropy.readthedocs.org/en/v0.3/api/astropy.coordinates.angles.Angle.html#astropy.coordinates.angles.Angle.wrap_at) intervals, and what I'm doing is flipping things if they're in the outside of (0, 180 degrees). I might be missing something obvious, but I don't think wrap_at can be used to solve this wrap issue.